scripts/src/ui/page/earnThankYou.tsx

```diff
@@ -5,11 +5,22 @@ import "../../../../styles/src/page/earnThankYou.styl";
 
 export interface EarnThankYouProps {
 	amount: number;
+	closeCb: () => void;
```

Comment:
decide whether you wanna name it `Cb` or `CB`.
also, I think that it's better to use `Handler`